### ISSUE TITLE
Floating point exception when running K-Quant model

### ISSUE THREAD

**Issue by nkoehring** [jmorganca/ollama#207]
I compiled ollama for Linux and want to use it with already downloaded models. I use models with llama.cpp CPU inference on my laptop, so they are all quantized anyway.

When trying with Mythologic with K-Quant (mythologic-13b.ggmlv3.q5_K_M.bin) method, I get a stack trace:

![image](https://github.com/jmorganca/ollama/assets/246402/28f0e195-8c53-4d6c-b7c9-f9b4293602ff)

**Comment by mchiang0610** [comment]
Thanks for submitting this. 

When I run it from main: 
```
./ollama run hi
>>> hi
Error: failed to load model
```

```
2023/07/25 11:15:41 routes.go:278: Listening on 127.0.0.1:11434
llama.cpp: loading model from /Users/mchiang/.ollama/models/blobs/sha256:8870d89fb916057519cf1fcb9ca6372d3215b767be524ca314a12ba104213e87
mOK/GԺA0��=Ny�w>�;[�zj#�Y�' should not be 13959180-dimensional
llama_load_model_from_file: failed to load model
[GIN] 2023/07/25 - 11:15:54 | 500 |  3.134388125s |       127.0.0.1 | POST     "/api/generate"


```

**Comment by mxyng** [comment]
resolved by #209
